### ISSUE TITLE
Host the helm chart

### ISSUE THREAD

**Issue by adriantr** [checkly/checkly-k8s#6]
Hey,

Do you have any plans on hosting the helm chart, so it's easily versioned and downloadable from the internet, without the need to package it manually?

**Comment by tnolet** [comment]
@adriantr can you expand a little on this? The helm charts are in this repo and versioned.

**Comment by tnolet** [comment]
@adriantr do you mean hosting our charts on https://artifacthub.io/?

**Comment by realmunk** [comment]
> @adriantr do you mean hosting our charts on https://artifacthub.io/?

Yes, that is what he was trying to say. Can you make them available through a registry so that we can avoid copying + pasting the chart?

**Comment by samortaylor** [comment]
Any update on this? Would be great if this helm chart was hosted like all other helm charts we use in our environment.

**Comment by tnolet** [comment]
> Any update on this? Would be great if this helm chart was hosted like all other helm charts we use in our environment.

Hi @samortaylor we have not implemented this yet. I did check out the options we have available. It's on the list and I will ping you once it's done
